.github/workflows/ci.yml

```diff
@@ -41,3 +41,4 @@ jobs:
 
     - name: Send report to Coveralls for package @ui5/linter
       uses: coverallsapp/github-action@v2.3.6
+      continue-on-error: true # Do not fail the job if coverage reporting fails (e.g. service is down)
```

Comment:
This one seems to be wrong. What I can find in the documentation is `fail-on-error` property: https://github.com/coverallsapp/github-action
Can you share where this config is taken from?